### PR TITLE
[IMPROVE] Rewrite Reactions Modal

### PR DESCRIPTION
**Pull request by dougfabris**
### before
image

### after
image

### CODE REVIEW

File: client/components/modals/ReactionList/ReactionList.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		}
+
+		openUserCard({
+			username,
+			rid,
+			target: e.currentTarget,
+			open: (e: React.MouseEvent<HTMLElement>) => {
+				e.preventDefault();
+				onClose();
+				tabBar.openUserInfo(username);
```
Comment (gabriellsh) [inline client/components/modals/ReactionList/ReactionList.tsx:36]: This tabBar is received as a prop of room. Shouldn't we update this and use "useTabBarOpenUserInfo" in place of this?

File: client/components/modals/ReactionList/Reactions.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+type ReactionsProps = {
+	reactions: ReactElement;
+	onClick: (e: React.MouseEvent<HTMLElement>) => void;
+};
+
+const Reactions = ({ reactions, onClick }: ReactionsProps): ReactElement => {
+	const useRealName = useSetting('UI_Use_Real_Name');
+	return (
+		<Box display='flex' flexDirection='column' minHeight='25vh'>
```
Comment (gabriellsh) [inline client/components/modals/ReactionList/Reactions.tsx:18]: I don't like this minHeight that much. If there are only a few reactions, it looks weird:
image

**Review by gabriellsh — CHANGES_REQUESTED** [review]

File: client/components/modals/ReactionList/Reactions.tsx (hunk 2 above)
Comment (dougfabris) [reply client/components/modals/ReactionList/Reactions.tsx:18]: changed to 15vh

File: client/components/modals/ReactionList/ReactionList.tsx (hunk 1 above)
Comment (dougfabris) [reply client/components/modals/ReactionList/ReactionList.tsx:36]: Already discussed

**Review by gabriellsh — APPROVED** [review]

**Pull request merged**